docker-compose/monitor/README.md

```diff
@@ -50,27 +94,27 @@ curl "http://localhost:16686/api/metrics/calls?service=driver&service=frontend&g
 ```
 
 
-## Example 2
+### Example 2
 Fetch P95 latencies for both the driver and frontend services from now,
 looking back 1 second with a sliding rate-calculation window of 1m and step size of 1 millisecond, where the span kind is either "server" or "client".
 
 ```bash
 curl "http://localhost:16686/api/metrics/latencies?service=driver&service=frontend&quantile=0.95&endTs=$(date +%s)000&lookback=1000&step=100&ratePer=60000&spanKind=server&spanKind=client" | jq .
 ```
 
-## Example 3
+### Example 3
 Fetch error rates for both driver and frontend services using default parameters.
 ```bash
 curl "http://localhost:16686/api/metrics/errors?service=driver&service=frontend" | jq .
 ```
 
-## Example 4
+### Example 4
 Fetch the minimum step size supported by the underlying metrics store.
 ```bash
 curl "http://localhost:16686/api/metrics/minstep" | jq .
 ```
 
-# HTTP API
+# HTTP API Specification
```

Comment:
I considered defining this as a swagger spec, but given its only "user" is Jaeger UI, I don't think it's worth the effort.